radar/platforms-and-services/azure-logic-apps.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Azure Logic Apps"
+ring: adopt
+quadrant: "platforms-and-aoe-services"
+tags: [cloud, azure, automation, integration]
+---
+
+[Azure Logic Apps](https://azure.microsoft.com/en-us/products/logic-apps/) is a
+cloud-based integration and automation service that enables building scalable
+workflows to orchestrate business processes and automate repetitive tasks. Logic
+Apps supports connectors for various cloud services, like Office 365, Dynamics
+365, and others, simplifying integration with on-premises applications and
+third-party services.
+
+### Use cases
+
+Here are some typical scenarios:
+
+- Business Process Automation: Create workflows to automate tasks like email
+  sending, request approvals, or updating records in enterprise databases.
+- System Integration: Connect and synchronize data across different systems,
+  such as CRM, ERP, and document management systems.
+- Monitoring and Management: Implement workflows to monitor events, send
+  real-time notifications, and manage automatic corrective actions.
+- Data Processing: Transform and transfer data between applications, performing
+  extract, transform, and load (ETL) operations.
+- Notification Management: Integrate with services like Microsoft Teams or Slack
+  to notify teams of important events or updates.
+
+### Reference of usage in our organization
+
+Azure Logic Apps is used within the IO infrastructure to orchestrate
+automated workflows and facilitate integration between internal systems and
+cloud services. Most configurations for Logic Apps in our organization have been
+managed directly through the Azure portal. Code-based configuration, however,
```

Comment:
Do you think we should update this statement to:
```suggestion
managed directly through the Azure portal (avoid this practice). Code-based configuration, however,
```

cc @gunzip

Reply:
I don't think so as this is a general advice that states for every other resource as well.

Reply:
Maybe we can suggest to at least backup (export) the appllicative in the codebase in case you've built it using the UI.